### PR TITLE
Create-example-pages

### PR DESCRIPTION
**Pull request by BleedingDev**

### CODE REVIEW

File: components/Audio.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import "vidstack/styles/defaults.css"
 import "vidstack/styles/community-skin/audio.css"
 
-import { alignClasses } from "@utils/puckUtils"
```
Comment (devodii) [inline components/Audio.tsx:4]: Any specific reason why you changed filename?

File: components/Embed.tsx [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+  height?: number
 }
 
 export function Embed({ src, title, width, height }: Props) {
   return (
     <iframe
-      referrerPolicy="no-referrer"
-      sandbox="allow-forms allow-scripts"
+      sandbox='allow-forms allow-scripts allow-same-origin'
+      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
```
Comment (devodii) [inline components/Embed.tsx:14]: What does this mean?

File: utils/puck/textPage.ts [hunk 3]
```diff
@@ -0,0 +1,136 @@
+export const textPage = {
```
Comment (devodii) [inline utils/puck/textPage.ts:1]: Why do we need this file? 
Something specific to puck config?

**Review by devodii — APPROVED** [review]

File: components/Audio.tsx (hunk 1 above)
Comment (BleedingDev) [reply components/Audio.tsx:4]: To group things about `puck` in one folder.

File: components/Embed.tsx (hunk 2 above)
Comment (BleedingDev) [reply components/Embed.tsx:14]: YouTube Embed was broken with `referrerPolicy` and I was also thinking about blocking of those things and if we are extremely strict, websites might break. So I've added this and later we'll figure out what is necessary and what is not. :)

File: utils/puck/textPage.ts (hunk 3 above)
Comment (BleedingDev) [reply utils/puck/textPage.ts:1]: This is one of the example pages. I didn't want to have one huge file with many example pages, so I created file per page. Bad thing is that we have to copy the output of the editor to the codebase (until we have DB).

**Pull request merged**